enos/enos-scenario-smoke.hcl

```diff
@@ -119,7 +122,7 @@ scenario "smoke" {
     }
 
     variables {
-      ami_id      = step.create_vpc.ami_ids[matrix.distro][matrix.arch]
+      ami_id      = step.create_vpc.ami_ids["ubuntu"]["amd64"]
```

Comment:
won't hardcoding here fail for other distro/arch ?

Reply:
The consul module doesn't support other architectures.